src/connect.js

```diff
@@ -20,7 +20,13 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  let result;
+
+  return (addObj) => {
```

Comment:
Good, but you can write simply 
```suggestion
  return (extraParams) => ({ ...params, ... extraParams });
```